lib/helper/WebDriverIO.js

```diff
@@ -668,6 +670,26 @@ class WebDriverIO extends Helper {
   }
 
   /**
+   * Return text from random element, if there is several elements with the same locator
+   */
+  grabTextFromRandomElement(locator) {
```

Comment:
The reason to add this method was that sometimes you have a large amount of similar elements and you want to test that they works as expected. But you don't want to test each element, because of time execution. Because of this you want to use any random element in your test.

Will remove this method.